docs/demand/transportation_electrification/data.rst

```diff
@@ -0,0 +1,39 @@
+Capabilities and Data
+#####################
+The transportation electrification module calculates an estimate of the additional hourly electricity demand from the electrification of transportation vehicles for all U.S. urban and rural areas for years ranging from 2017-2050.  The hourly estimation builds upon data collected representing over half a million driving events. 
+
+Additional data sets allow for profiles to be generated across the following
+dimensions: 
+
++ 4 vehicle types (light-duty vehicles (LDV), light-duty trucks (LDT), medium-duty
+  vehicles (MDV), and heavy-duty vehicles (HDV)) 
++ 33 simulation years (2017-2050) 
```

Comment:
@BainanXia @danlivengood I don’t think a year range is right here?Rather four specific years:
```
:param int projection_year: year that is being modelled/projected to, 2017, 2030, 2040, 2050.
```

Reply:
@dmuldrew technically you are correct at the moment.  However, what I want to explore via the Regional_scaling_factor csv creation process is whether NREL has projections for all years from 2017-2050 or just those four.  I believe they have all 33, and the only reason we only have 4 is that's the subset that Brian wrote up in Matlab.  With the Python process, assuming the data exists, we could do all 33.

Reply:
Keep it as is? We can go through it in the final stage to make sure everything is correct.

Reply:
I say yes.

Reply:
Fyi, confirmed that NREL's data is indeed all 33 years:
"Scenario data from the Electrification Futures Study Scenarios of Electric Technology Adoption and Power Consumption for the United States report. Annual projections from 2017 to 2050 of electric technology adoption and energy consumption for five scenarios"
from https://data.nrel.gov/submissions/92